src/fswAlgorithms/effectorInterfaces/thrusterPlatformReference/thrusterPlatformReference.rst

```diff
@@ -97,12 +108,15 @@ The module is configurable with the following parameters:
     * - ``r_FM_F``
       - [0, 0, 0]
       - relative position of point :math:`F` with respect to point :math:`M`, in :math:`\mathcal{F}`-frame coordinates
-    * - ``r_TF_F``
-      - [0, 0, 0]
-      - relative position of point :math:`T` with respect to point :math:`F`, in :math:`\mathcal{F}`-frame coordinates
-    * - ``T_F``
-      - [0, 0, 0]
-      - thrust vector in :math:`\mathcal{F}`-frame coordinates
     * - ``K``
       - 0
       - proportional gain of the momentum dumping control loop
+    * - ``Ki``
```

Comment:
You should say this is an optional parameter.

Reply:
Done.